qa/rpc-tests/wallet_accounts.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+# Copyright (c) 2022 The Zcash developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or https://www.opensource.org/licenses/mit-license.php .
+
+from test_framework.authproxy import JSONRPCException
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import (
+    assert_equal,
+    assert_raises_message,
+    get_coinbase_address,
+    start_nodes,
+    wait_and_assert_operationid_status,
+)
+
+from decimal import Decimal
+
+# Test wallet accounts behaviour
+class WalletAccountsTest(BitcoinTestFramework):
+    def setup_nodes(self):
+        return start_nodes(self.num_nodes, self.options.tmpdir, [[
+            '-experimentalfeatures',
+            '-orchardwallet',
+        ]] * self.num_nodes)
+
+    def check_receivers_exist(self, ua, expected):
```

Comment:
The naming of this method suggests that it is checking just that all the expected receiver types exist, i.e. that the actual set is a superset of `expected`. It is in fact checking that the set is exactly the same. That is what the tests need, but a better name would be something like `check_receiver_types`.